modules/tinymce/src/core/test/ts/browser/newline/InsertNewLineTest.ts

```diff
@@ -1662,4 +1662,96 @@ describe('browser.tinymce.core.newline.InsertNewLineTest', () => {
       });
     });
   });
+
+  context('TINY-12073: nested inline elements', () => {
```

Comment:
You can skip the ticket ID on the context since if we find a bug in this code then later we add a new ticket and add that to the context. Specific `it` calls can still have the ticket id.